SportsHub/SportsHub.DAL/Data/Configurations/ArticleEntityTypeConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using SportsHub.Domain.Models;
+
+namespace SportsHub.DAL.Data.Configurations
+{
+    public class ArticleEntityTypeConfiguration : IEntityTypeConfiguration<Article>
+    {
+        public void Configure(EntityTypeBuilder<Article> article)
+        {
+            article.Property(x => x.Title)
+                .IsRequired(true)
+                .HasMaxLength(100)
+                .IsUnicode(true);
+
+            article.Property(x => x.Content)
+                .IsRequired(true)
+                .IsUnicode(true);
```

Comment:
why in some places IsUnicode set as true and in others as False?

Reply:
Unified those.